docroot/modules/custom/foia_annual_data_report/foia_annual_data_report.module

```diff
@@ -0,0 +1,92 @@
+<?php
+
+/**
+ * @file
+ * Primary module hooks for foia_annual_data_report module.
+ *
+ */
+
+use Drupal\Core\Form\FormStateInterface;
+use Drupal\views\Views;
+use Drupal\Core\Ajax\AjaxResponse;
+use Drupal\Core\Ajax\AlertCommand;
+
+/**
+ * Implements hook_form_form_id_alter().
+ */
+function foia_annual_data_report_form_node_annual_foia_report_data_edit_form_alter(&$form, FormStateInterface $form_state, $form_id) {
+  foia_annual_data_report_ajax_component_to_agency($form, $form_state, $form_id);
```

Comment:
This alter function is used. I do not think the other one is, at least not at `/node/{nid}/edit`.